src/main.cpp

```diff
@@ -434,6 +434,7 @@ static string _get_oauth_token(const string &device_uuid)
 	data = "grant_type=urn:ietf:params:oauth:grant-type:device_code";
 	data += "&device_code=" + json.get<string>("device_code");
 	data += "&client_id=" + device_uuid;
+	data += "&scopes=" + factory + ":devices:create";
```

Comment:
Hey @mike-sul, just a minor thing: here the parameter needs to be `scope` (singular, as per oauth spec). The plural version `scopes` is what is returned when the oauth token is used to do auth/auth.

In the singular form is a string (with space separated values), in the plural form it will be an array of strings.

Reply:
And apologies, it was me in chat that said to use `scopes`, I thought you were referring to the response part, not the request.

Reply:
NP :), just updated